pd-client/client.go

```diff
@@ -344,11 +345,20 @@ func (c *client) tsLoop() {
 }
 
 func (c *client) processTSORequests(stream pdpb.PD_TsoClient, requests []*tsoRequest) error {
+	opts := make([]opentracing.StartSpanOption, 0, len(requests))
```

Comment:
I guess this may hurt the performance because TSO is a too frequent operation.

Do you run tso-benchmark?

Reply:
I've make a tiny change, so that if tracing is not used,  there will be zero cost.

Reply:
if we enable this, how much QPS will we reduce?

Reply:
Seems 20% under high pressure in benchmark, it introduce some object allocation here.